scripts/generate/generate-src.ts

```diff
@@ -34,9 +34,11 @@ export function generateSrc(Template: Template) {
     const templateSource = getTemplateSrc(Template);
     fs.writeFileSync(outPath, templateSource);
 
-    console.log(`src/index${extension} file was created.`);
+    copyAdditionalFiles();
```

Comment:
🤔  The naming especially the word `additional` doesn't seem very meaningful. Perhaps it would be better to indicate what kind of files? Since it's taking from `TemplateRootDirectory`, maybe `copyTemplateFiles` ?

Reply:
Thought about that, but we have files inside a `template` folder and I thought that might be confusing as we are *not* copying these. 🤔

Reply:
Ah. If there's any commonality to the files copied now and in the future, it would be nice to use it for the naming. At the end of the day though, it's not super important.

Reply:
`copyFiles` ? :D

Reply:
Good enough for now 😄